.github/workflows/ci.yml

```diff
@@ -107,14 +108,15 @@ jobs:
     with:
       build-ctk-ver: ${{ needs.ci-vars.outputs.CUDA_BUILD_VER }}
 
-  checks:
-    name: Check job status
-    permissions:
-      checks: read
-    needs:
-      - build
-      - test-linux
-      - test-windows
-      - doc
-    secrets: inherit
-    uses: ./.github/workflows/status-check.yml
+  # Temporarily disabled status check job as it depends on test jobs
+  # checks:
+  #   name: Check job status
+  #   permissions:
+  #     checks: read
+  #   needs:
+  #     - build
+  #     - test-linux
+  #     - test-windows
+  #     - doc
+  #   secrets: inherit
+  #   uses: ./.github/workflows/status-check.yml
```

Comment:
Commenting out this block is overaggressive. Just comment out the two lines test-Linux and test-windows.